docs/KafkaFile.md

```diff
@@ -0,0 +1,34 @@
+# Kafka file
+
+A Kafka file is a file with `.kafka` file extension which provides the capability to declare:

```

Comment:
A Kafka file is a file with  the `.kafka` file extension. It provides, for the currently selected cluster, the ability to declare:

Reply:
fixed